config/releases.exs

```diff
@@ -31,6 +31,15 @@ defmodule Environment do
       _ -> nil
     end
   end
+
+  def get_local_url(key) do
+    url = get(key)
+
+    case URI.parse(url).host do
+      host when host in ~w(0.0.0.0 127.0.0.1 localhost host.docker.internal) -> url
+      host -> raise "Expected host of the #{key} environment variable to be `localhost` or `127.0.0.1`, got: #{host}"
```

Comment:
Maybe we could use a varaible for  `~w(0.0.0.0 127.0.0.1 localhost host.docker.internal)` and use it to be more precise in the raise message exception

Reply:
True, it’s a little misleading to provide two valid values when we actually have four valid ones 😄